shared/gh/api/gh.api.util.js

```diff
@@ -96,7 +96,13 @@ define(['exports', 'moment', 'bootstrap-notify'], function(exports, moment) {
         }
 
         // TODO: add the week number when the custom configuration is in place
-        var weekNumber = 'W?';
+        // var weekNumber = 'W' + getWeekInTerm(startDate);
+        var weekNumber = getWeekInTerm(startDate);
+        if (weekNumber === 0) {
+            weekNumber = 'OT';
```

Comment:
I've been talking to Oszkar and this is the expected text for events that fall out of term (hence OT)